services/alerts/src/AlertsProvider.tsx

```diff
@@ -5,15 +5,15 @@ import { makeAlertsManager } from './makeAlertsManager'
 import type { Alert, AlertsManager } from './types'
 
 export const AlertsProvider = ({
-    plugin,
-    parentAlertsAdd,
-    showAlertsInPlugin,
     children,
+    plugin = false,
+    parentAlertsAdd = undefined,
```

Comment:
It's certainly not necessary to provide these default values, not sure what our team thinks is good good practice here?
I like seeing default values, but that's personal / preference.

Reply:
Do you also want to add a default `false` for `showAlertsInPlugin`?